src/edgepi/eeprom/README.md

```diff
@@ -0,0 +1,57 @@
+# EdgePi EEPROM Module
+EEPROM module is used to store data to on-board EEPROM via I2C protocol
+
+# Hardware
+The on-board EEPROM provide 32768 Bytes of data. The available memory is divided into two sections.
+1. EdgePi Reserved Space, 0~16383 Bytes
+2. User Space, 16384~32767 Bytes
+
+Each section has separate methods to read/write. 
+
+__It is not recommanded for the end user to write to EdgePi Reserved Space__
+
+# Example Code
+
+## Reading EdgePi Reserved Memory
+```python
+from edgepi.eeprom.edgepi_eeprom import EdgePiEEPROM
+
+edgepi_eeprom = EdgePiEEPROM()
+eeprom_data = edgepi_eeprom.read_edgepi_data()
+```
+## Writing EdgePi Reserved Memory
+__NOTE__: To write to the eeprom memory, the memory needs to be read first.
+```python
+from edgepi.eeprom.edgepi_eeprom import EdgePiEEPROM
+from edgepi.eeprom.eeprom_constants import EepromModuleNames
+
+edgepi_eeprom = EdgePiEEPROM()
+# Read the eeprom memory first
+eeprom_data = edgepi_eeprom.read_edgepi_data()
+# Changing Config_key certificate
+eeprom_data.config_key.certificate = "Changed Config key certificate"
+# Write the modified dataclass to the memory
+edgepi_eeprom.write_edgepi_data(eeprom_data)
+```
+
+## Reset EdgePi Reserved Memory
+```python
+from edgepi.eeprom.edgepi_eeprom import EdgePiEEPROM
+from edgepi.eeprom.eeprom_constants import EepromModuleNames, DEFAULT_EEPROM_BIN_B64
```

Comment:
do we need `EepromModuleNames` ?

Reply:
[9369784](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/357/commits/93697841e4024409e0df435121ce59cd163fcc86)